dezoomers/vls.js

```diff
@@ -0,0 +1,50 @@
+var vls = (function() {
+  return {
+    name: 'VLS',
+    description: 'Visual Library Server, by semantics',
+    urls: [
+      /\/(thumbview|pageview|zoom)\/\d+$/
```

Comment:
Maybe this url matcher is a little too permissive... We'll see if we get complaints about this dezoomer being chosen by error. This dezoomer will be the last to be tested when searching for a dezoomer anyway.